### DISCUSSION TITLE
RTL Support, Integrate with Metronic v9 template

### DISCUSSION THREAD

**Discussion by vhdm** [livewire/flux#926] [Feature Requests]
Hello👋, thank you so much for your **beautiful** and **user-friendly** FluxUI package. 

How can I use this package on **RTL** websites?

Is it possible to integrate it with the **[Metronic v9](https://keenthemes.com/metronic/tailwind/demo1/)** template, which is styled with Tailwind CSS? How can this be done?

**Comment by joshhanley** [comment]
@vhdm please make sure to open separate issues/ feature requests then putting everything in one, as if one get's implemented but not the other, we cannot mark it as complete.

Flux currently doesn't have RTL support. There is already an open feature request for it here #196, so going to close this one.

Regarding your other question, we've not heard of Metronic, so sorry can't help you there. You will need to experiment to see if it will work.
